scripts/linters/pylint_extensions.py

```diff
@@ -830,7 +830,8 @@ def check_docstring_structure(self, node):
         if node.doc:
             docstring = node.doc.splitlines()
             # Check for space after """ in docstring.
-            if docstring[0][0] == b' ':
+            # The first check below helps avoiding empty docstring situation.
```

Comment:
I think you can remove this comment because the above comment already explains what you are trying to do here, and it's a good practice to **avoid** writing comments for "how" we are doing it (code itself should be self explanatory, which I think it already is here).